docs/tutorials/annotation/annotate_points.md

```diff
@@ -97,14 +101,17 @@ def point_annotator(
     """
     stack = imread(im_path)
 
+    fake_data = [[0.0, 0.0, 0.0]] * len(labels)
```

Comment:
Without this fake data `add_points` fails for me with this error:
```
  File "/home/pape/software/conda/miniconda3/envs/sam/lib/python3.10/site-packages/napari/components/viewer_model.py", line 4, in add_points
    import itertools
  File "/home/pape/software/conda/miniconda3/envs/sam/lib/python3.10/site-packages/napari/layers/points/points.py", line 369, in __init__
    self._feature_table = _FeatureTable.from_layer(
  File "/home/pape/software/conda/miniconda3/envs/sam/lib/python3.10/site-packages/napari/layers/utils/layer_utils.py", line 956, in from_layer
    features = _features_from_properties(
  File "/home/pape/software/conda/miniconda3/envs/sam/lib/python3.10/site-packages/napari/layers/utils/layer_utils.py", line 1024, in _features_from_properties
    return _validate_features(properties, num_data=num_data)
  File "/home/pape/software/conda/miniconda3/envs/sam/lib/python3.10/site-packages/napari/layers/utils/layer_utils.py", line 999, in _validate_features
    return pd.DataFrame(data=features, index=index)
  File "/home/pape/software/conda/miniconda3/envs/sam/lib/python3.10/site-packages/pandas/core/frame.py", line 709, in __init__
    mgr = dict_to_mgr(data, index, columns, dtype=dtype, copy=copy, typ=manager)
  File "/home/pape/software/conda/miniconda3/envs/sam/lib/python3.10/site-packages/pandas/core/internals/construction.py", line 481, in dict_to_mgr
    return arrays_to_mgr(arrays, columns, index, dtype=dtype, typ=typ, consolidate=copy)
  File "/home/pape/software/conda/miniconda3/envs/sam/lib/python3.10/site-packages/pandas/core/internals/construction.py", line 120, in arrays_to_mgr
    arrays, refs = _homogenize(arrays, index, dtype)
  File "/home/pape/software/conda/miniconda3/envs/sam/lib/python3.10/site-packages/pandas/core/internals/construction.py", line 608, in _homogenize
    com.require_length_match(val, index)
  File "/home/pape/software/conda/miniconda3/envs/sam/lib/python3.10/site-packages/pandas/core/common.py", line 576, in require_length_match
    raise ValueError(
ValueError: Length of values (2) does not match length of index (0)
```
(This is for a case where I have 2 label values.)

Reply:
Ah, the issue is that `properties` is deprecated, and now you want to use `features` (when you have a table) and `feature_defaults` (when you just want to give options, as below). Can you try that? ie remove the fake data, change `properties=` to `feature_defaults=`?

Reply:
I tried, but `feature_defaults` is currently not available via `add_points`, see https://github.com/napari/napari/issues/5789.

Reply:
They are available on main branch and this repository contains code for the current develop branch. We split docs from napari/napari after 0.4.17 release.

Reply:
> They are available on main branch and this repository contains code for the current develop branch. We split docs from napari/napari after 0.4.17 release.

I see. I was under the impression that the docs here reference the stable releases. I have changed my set-up and I am now checking with the most up-to-date `main` branch of napari.

One more question though: what do you mean with 
> contains code for the current develop branch

I don't see a `develop` branch in https://github.com/napari/napari.